packages/newspringchurchapp/src/ui/ContentCardConnected/query.js

```diff
@@ -33,23 +33,23 @@ export const BASE_CARD_FRAGMENT = gql`
     title
     hyphenatedTitle: title(hyphenated: true)
     summary
-    isLiked
     ... on MediaContentItem {
       videos {
         sources {
           uri
         }
       }
+      parentChannel {
+        id
+        name
+      }
     }
     ... on WeekendContentItem {
       videos {
         sources {
           uri
         }
       }
-      liveStream {
-        isLive
-      }
       parentChannel {
```

Comment:
probably should delete it from down here since core added it above

Reply:
I think `parentChannel` is on each of the "content items" now. https://github.com/ApollosProject/apollos-prototype/blob/master/packages/apolloschurchapp/src/ui/ContentCardConnected/query.js#L53